src/components/Modal/TbtcRecoveryFileModal/index.tsx

```diff
@@ -38,11 +43,14 @@ const TbtcRecoveryFileModalModal: FC<
   }
 
   const handleDownloadClick = (data: DepositScriptParameters) => {
-    downloadFile(
-      JSON.stringify(data),
-      "deposit-script-parameters.json",
-      "text/json"
-    )
+    const date = new Date()
+    const fileName = `
+    ${ethAddress}-
+    ${btcDepositAddress}-
+    ${date.getDate()}/${date.getMonth() + 1}/${date.getFullYear()}
```

Comment:
New lines makes the file name weird (so many spaces). Let's move it to one line and format the code. It should look something like this:

```suggestion
    const fileName = `${ethAddress}-${btcDepositAddress}-${date.getDate()}/${
      date.getMonth() + 1
    }/${date.getFullYear()}`
```

Reply:
d79cc94